app/components/provider_interface/completed_offer_summary_component.rb

```diff
@@ -10,13 +10,15 @@ def rows
           value: course_option.provider.name_and_code,
           action: {
             href: change_provider_path,
+            visually_hidden_text: 'training provider',
```

Comment:
Could you point me to the ticket this change relates to?

Reply:
Under Examples of additional instances subheading, DAC_Non_Descriptive_Links_02. References are not a row on that page, so I took it to mean that all the change links on `provider/applications/{id}/offers` were lacking text for the screen reader.

Reply:
Good to keep this I think. Ideally we'd somehow do a search across the whole codebase to make sure that all of the 'Change' links have visually-hidden text like this, in case DAC missed any...

Reply:
yea def keep, I just thought I missed something in the tickets